nautobot/extras/api/views.py

```diff
@@ -461,9 +461,12 @@ def _run_job(request, job_model, legacy_response=False):
     if not job_model.installed:
         raise MethodNotAllowed(request.method, detail="This job is not presently installed and cannot be run")
     if job_model.has_sensitive_variables:
-        if request.data.get("schedule") and request.data["schedule"]["interval"] != JobExecutionType.TYPE_IMMEDIATELY:
+        if (
+            request.data.get("scheduled_job")
+            and request.data["scheduled_job"]["interval"] != JobExecutionType.TYPE_IMMEDIATELY
```

Comment:
This is potentially flaky with malformed request data, perhaps:

```suggestion
            "scheduled_job" in request.data
            and "interval" in request.data["scheduled_job"]
            and request.data["scheduled_job"]["interval"] != JobExecutionType.TYPE_IMMEDIATELY
```